website/content/partials/known-issues/config_listener_proxy_protocol_behavior_issue.mdx

```diff
@@ -0,0 +1,33 @@
+<a id="listener-proxy-protocol-config" />
+
+### Potential DoS when using the `deny_unauthorized` proxy protocol behavior for a TCP listener
+
+#### Affected versions
+
+##### Community Edition (CE)
+
+- 1.10.x - 1.15.x
+- 1.17.1
+
+##### Enterprise
+
+- 1.10.x+ent - 1.15.11+ent
+- 1.16.5+ent
+- 1.17.1+ent
+
+#### Issue
+
+Vault TCP listeners configured with the `deny_unauthorized` [proxy_protocol_behavior](/vault/docs/configuration/listener/tcp#proxy_protocol_behavior)
+which receive a request from an untrusted upstream connection will close, causing
+Vault to no longer respond to requests via that listener.
+
+This leaves the Vault API vulnerable to a denial-of-service (DoS) attack by requests
```

Comment:
What is "this"? The closed listener or the fact that Vault can't receive requests through it?